test/FastTests/Corax/Ranking/QuantizationTest.cs

```diff
@@ -25,4 +27,23 @@ public void CanEncodeAndDecodeEveryNumberUnderShort()
             Assert.False(decoded <= 0);
         }
     }
+    
+    [Theory]
+    [InlineData(7)]
+    [InlineData(8)]
+    [InlineData(16)]
+    [InlineData(24)]
+    [InlineData(32)]
+    [InlineData(33)]
+    [InlineData(1)]
+
+    public void SimdInstructionCorrectlyIgnoresFrequency(int size)
+    {
+        var random = new Random(2337);
+        var ids = Enumerable.Range(0, size).Select(i => (long)random.Next(31_111, 59_999)).ToArray();
+        var idsWithShifted = ids.Select(i => i << 10).ToArray();
+        
+        EntryIdEncodings.DecodeAndDiscardFrequencySimd(idsWithShifted.AsSpan(), size);
+        Assert.Equal(ids, idsWithShifted);
+    }
```

Comment:
Add also a fuzzying test for equality of SIMD and Classic implementation.

Reply:
Added 👍